docs/src/main/sphinx/connector/kafka.md

```diff
@@ -433,7 +433,7 @@ Inserts are not supported, and the only data format supported is AVRO.
         * ``IGNORE`` - Ignore structs with no fields. This propagates to parents.
           For example, an array of structs with no fields is ignored.
         * ``FAIL`` - Fail the query if a struct with no fields is defined.
-        * ``DUMMY`` - Add a dummy boolean field called ``dummy``, which is null.
+        * ``DUMMY`` - Add a dummy boolean field name consisting of random characters ``$empty_field_marker``, which is null.
```

Comment:
So now, should replace `DUMMY` to something more meaningful? Maybe `MARK`?

Reply:
nice catch, thanks!